cert-manager/overlays/azuredns/secret-certmanagersecret-certmanager.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cert-manager-secret
+  namespace: cert-manager
+type: Opaque
+data:
+  client-secret: $base64(certManagerAzurednsSpPassword)
```

Comment:
@Vafilor why use `data` here and then do `$base64`? Doesn't `stringData` already take care of this like the `route53` implementation?

Reply:
Referring to this https://github.com/onepanelio/manifests/blob/feature/dns/cert-manager/overlays/route53/secret-certmanagersecret-certmanager.yaml#L8

Reply:
I believe it does. I need to test that part with the generated yaml. I ran into an issue somewhere with the data not being encoded and may have put it here as well.